lib/github.com/mpllang/mpllib/ParFuncArray.sml

```diff
@@ -100,7 +100,7 @@ struct
             i
             *)
       in
-        if #1 ((*valOf*) (Array.sub (data, n-1))) < v then NONE else
+        if #1 (*valOf*) (Array.sub (data, n-1)) < v then NONE else
```

Comment:
The parens here aren't really unnecessary; the commented-out `valOf` would need them if re-enabled.